src/moustachu.nim

```diff
@@ -176,12 +181,12 @@ proc render(tmplate: string, contextStack: seq[Context]): string =
         renderings.add(indentation)
 
     of TokenType.partial:
-      var partialTemplate = contextStack.lookupString(token.value)
+      var partialTemplate = pwd.joinPath(token.value & ".mustache").readFile
```

Comment:
I still like to be able to pass in-memory moustache strings in my context variables. And those should take precedence over disk partials.

We can have something like:

```nim
var partialTemplate = contextStack.lookupString(token.value)
if partialTemplate == "":
  partialTemplate = readFile(partials_dir.joinPath(token.value & ".mustache"))
```